src/features/embalm/index.tsx

```diff
@@ -9,7 +9,7 @@ import { networkConfigs } from 'lib/config/networkConfig';
 
 export function Embalm() {
   useLoadArchaeologists();
-  useBootLibp2pNode();
+  useBootLibp2pNode(20_000);
```

Comment:
moved the discovery period to be an arg -- this is undefined for the arch list page, as want constant discovery for that page.

note: currently if you visit the archaeologist list *before* embalming, then peer discovery will be indefinite. Could spend some time handling this case (resetting that value), but since this is test functionality currently (and that discovery period will be removed anyways eventually), figured it wasn't worth the effort.